extension/scripts/features/armory-worth/ttArmoryWorth.js

```diff
@@ -91,6 +91,9 @@
 	}
 
 	function removeWorth() {
-		document.findAll(".tt-armory-worth").forEach((x) => x.remove());
+		document.findAll(".tt-armory-worth").forEach((x) => {
+			x.classList.remove("tt-modified");

```

Comment:
Why would you remove a class on an element that you are removing immediately after?